### PR TITLE
Tag JPLEphemeris v0.3.0

### PR DESCRIPTION
**Pull request by helgee**

### CODE REVIEW

File: JPLEphemeris/versions/0.3.0/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4-
```
Comment (tkelman) [inline JPLEphemeris/versions/0.3.0/requires]: the trailing dash refers to prereleases of 0.4, do you still need to support those?

**Comment by helgee** [comment]
@tkelman Good catch. How do I fix this? Do I need to open a new PR?

**Comment by tkelman** [comment]
No, you can just delete the trailing `-` in place at https://github.com/helgee/METADATA.jl/blob/pull-request/c30e0d65/JPLEphemeris/versions/0.3.0/requires, just be sure to make the same change in the package's `REQUIRE` file too before the next tag

**Comment by helgee** [comment]
Done. Thanks!

**Pull request merged**